client/control/log/log_control.go

```diff
@@ -0,0 +1,179 @@
+package logcontrol
+
+import (
+	"net/url"
+
+	"github.com/go-kit/kit/endpoint"
+	"golang.org/x/net/context"
+
+	"github.com/xh3b4sd/anna/api"
+	"github.com/xh3b4sd/anna/server/control/log"
+	serverspec "github.com/xh3b4sd/anna/server/spec"
+)
+
+type Config struct {
+	URL *url.URL
+}
+
+func DefaultConfig() Config {
+	newConfig := Config{
+		URL: &url.URL{
+			Host:   "127.0.0.1:9119",
+			Scheme: "http",
+		},
+	}
+
+	return newConfig
+}
+
+func NewLogControl(config Config) serverspec.LogControl {
```

Comment:
There should be tests for that like for the text interface client implementation.